addons/web/static/src/views/fields/many2one/many2one_field.js

```diff
@@ -318,7 +318,7 @@ export class Many2OneField extends Component {
         if (records.length === 1) {
             this.update([{ id: records[0].id, name: records[0].name }]);
         } else {
-            const searchInput = this.autocompleteContainerRef.el.querySelector("input");
+            const searchInput = this.autocompleteContainerRef.el?.querySelector("input");
```

Comment:
this line of code is helping a lot. the L109 was the problematic code. I can revert this LOC and it was because there is two lines calling `this.autocompleteContainerRef.el` and I give the `?,` to all of them.